.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server
+    We appreciate your time and interest in helping our project!
+
+    Use this template to help us review your change. Not everything is required,
+    depending on your change. Keep or delete what is relevant for your change.
+    Remember that it helps us review if you give more helpful info for us to
+    understand your change.
```

Comment:
"Remember that the more helpful info your pull request includes, the easier it is for us to 
    understand and review your changes."
Maybe?

Reply:
Same as above, this is from [Picard’s PR template](https://github.com/metabrainz/picard/blob/master/.github/PULL_REQUEST_TEMPLATE.md), thus I requested a review from @zas

Reply:
@reosarevok's suggestion sounds better here

Reply:
Changed by https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/3edc2d4bf5caf3712a06fb30c7297a24249c738c